installers/MacInstaller.hs

```diff
@@ -72,13 +72,19 @@ main = do
        ]
   run "productbuild" productargs
 
-  -- Sign the installer with a special macOS dance
-  run "security" ["create-keychain", "-p", "travis", "macos-build.keychain"]
-  run "security" ["default-keychain", "-s", "macos-build.keychain"]
-  shells "security import macos.p12 -P $CERT_PASS -k macos-build.keychain -T `which productsign`" mempty
-  run "security" ["set-key-partition-list", "-S", "apple-tool:,apple:", "-s", "-k", "travis", "macos-build.keychain"]
-  run "security" ["unlock-keychain", "-p", "travis", "macos-build.keychain"]
-  shells ("productsign --sign \"Developer ID Installer: Nikolaos Bentenitis\" --keychain macos-build.keychain dist/temp2.pkg " <> T.pack pkg) mempty
+  isPullRequest <- fromMaybe "false" <$> lookupEnv "TRAVIS_PULL_REQUEST"
```

Comment:
We should only sign when `$TRAVIS_PULL_REQUEST` is defined and is `"false"`. If for some strange reason `$TRAVIS_PULL_REQUEST` was ever undefined this code would attempt to sign anyway.

Please change the default case to something other than "false".